src/chart/generateCategoricalChart.js

```diff
@@ -76,6 +76,7 @@ const generateCategoricalChart = ({
       onMouseMove: PropTypes.func,
       onMouseDown: PropTypes.func,
       onMouseUp: PropTypes.func,
+      reverseGroupOrder: PropTypes.bool,
```

Comment:
@jayseeg  How about  `reverseStackOrder` ?

Reply:
👍 —that's actually what I started with,... can't recall the reason for the change